src/server/game/Spells/SpellMgr.cpp

```diff
@@ -2821,6 +2822,10 @@ void SpellMgr::LoadSpellInfoCustomAttributes()
                 case SPELL_AURA_POWER_BURN:
                     spellInfo->AttributesCu |= SPELL_ATTR0_CU_NO_INITIAL_THREAT;
                     break;
+                case SPELL_AURA_PERIODIC_DAMAGE:        // DoTs like Shadow Word: Pain
+                    spellInfo->AttributesCu |= SPELL_ATTR0_CU_NO_INITIAL_THREAT;
```

Comment:
What do you mean ? This flag is assigned on master, I did not make this up. I only added the SPELL_ATTR0_CU_AURA_CC flag.